src/retrace/retrace.py

```diff
@@ -1678,6 +1690,10 @@ def get_vmlinux(self):
     def set_vmlinux(self, value):
         self.set(RetraceTask.VMLINUX_FILE, value)
 
+    def has_vmcore(self):
```

Comment:
Could you make this work with coredump as well?
It is the same as `vmcore`, but is called `coredump`.  Maybe rename function to `has_core` and check both, or when you check that if `has_vmcore` check if `has_vmcore or has_coredump`.

Reply:
I think a 'has_coredump' is better.  Unfortunately I do not have the ability to test userspace cores because that has never worked correctly on our instances but I can put another patch on this series.